src/deploy.js

```diff
@@ -94,6 +94,11 @@ module.exports = function deploy() {
           type: 'boolean',
           default: false,
         })
+        .option('content', {
```

Comment:
why not `contentRepo` like in helix-config.yaml ?

Reply:
Because camel case command line arguments are really uncommon.